### ISSUE TITLE
Question 2

### ISSUE THREAD

**Issue by kartset** [CShorten/KaggleDogBreedChallenge#2]
![error1](https://user-images.githubusercontent.com/30878021/56559529-a2883200-65ed-11e9-950a-9cda6fb6801b.png)

Can you explain why am I getting these warning and what should I do to eradicate them and what are the possible errors that I might get due to them?

![error 2](https://user-images.githubusercontent.com/30878021/56559702-090d5000-65ee-11e9-83b5-42cb5120de3d.png)

And is this error related to above warning in any way and how to solve this ?

Thanks

**Comment by danucalovj** [comment]
Same here :(

**Comment by danucalovj** [comment]
@kartset try this:

Change this:

`model.compile(loss='binary_crossentropy', optimizer='adam', metrics = ['accuracy'])`

To this:

`model.compile(loss='sparse_categorical_crossentropy', optimizer='adam', metrics = ['accuracy'])`

Not sure of results yet, running epochs right now, at least got it to run.

**Comment by danucalovj** [comment]
Nevermind. That didn't work.

**Comment by Micajah-J** [comment]
I think it's something to do with the object trainLabels. If you look at the line of code that creates the object, it looks like they wanted to create a for loop but didn't complete it so the line only returns an array full of emptiness.
